app/code/Magento/GroupedProduct/view/base/templates/product/price/final_price.phtml

```diff
@@ -26,7 +26,7 @@ if ($minProduct) {
         );
 }
 ?>
-<div class="price-box" itemprop="offers" itemscope itemtype="http://schema.org/Offer">
+<div class="price-box">
```

Comment:
I'm not expert about the topic but looking at http://schema.org/Offer specification I wonder whether the `price-box` element should have `price` and `priceCurrency` specified rather than nothing.

Let me know, thank you!

Reply:
Today in the "Starting at" part. But here, you don't have a product you can sell. That why we have to remove it.
We should have it in the list of available products (table-wrapper grouped), 3 times for the example product. I'll make another PR later to add it here, I'm still waiting validation from my SEO guy.